client/verta/tests/unit_tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""Pytest fixtures for use in client unit tests."""
+
+import os
+from unittest.mock import patch
+
+import pytest
+
+from verta._internal_utils._utils import Connection, Configuration
+from verta.credentials import EmailCredentials
+
+
+@pytest.fixture(scope="session")
+def mock_conn() -> Connection:
+    """ Return a mocked object of the _internal_utils._utils.Connection class for use in tests """
+    with patch.dict(os.environ, {'VERTA_EMAIL': 'test_email@verta.ai', 'VERTA_DEV_KEY':'123test1232dev1232key123'}):
+        credentials = EmailCredentials.load_from_os_env()
```

Comment:
I changed this from a decorator on the function to a context here, because I'm not sure whether patching `os.environ` (which as far as I know is global) on a session-scoped fixture would affect other tests.

Reply:
I see no downside to this approach.  We only need those env vars in place to create the fixture once anyway.